data/gallery.js

```diff
@@ -176,66 +195,85 @@ function removeNextButt() {
   }
   if (
     currentImageIndex ===
-    galleryAlbums[currentAlbumIndex].images.length - 1
+    galleryAlbums[currentAlbumIndex].fields.images.length - 1
   ) {
     forwardButt.style.visibility = 'hidden';
   } else {
     forwardButt.style.visibility = 'visible';
   }
 }
 
+// sets the single image file location
 function updateSingleImage() {
-  singleImage.src =
-    galleryAlbums[currentAlbumIndex].images[currentImageIndex].src;
+  singleImage.src = `https://${galleryAlbums[currentAlbumIndex].fields.images[currentImageIndex].fields.file.url}`;
   removeNextButt();
```

Comment:
i think contentful returns the url with the preceeding slashes, i.e.`//assets.cf.com/.....` so i dont think you need the `https://` rather it could be `https:`

(and if it wasn't being used locally you could even leave off the https: as per what we've discussed previously)